tools/tools.qmd

```diff
@@ -1,187 +1,93 @@
-  </tr>
-  <tr>
-    <td></td>
-    <td>[Highlight.js: a syntax highlighter written in JavaScript](https://github.com/highlightjs/highlight.js){target="_blank"} - [(Source code)](https://github.com/highlightjs/highlight.js/blob/main/src/languages/stan.js){target="_blank"}</td>
-  </tr>
-  <tr>
-    <td>latex</td>
-    <td>[lstbayes: LaTeX Listings language drivers for BUGS, JAGS, and Stan](https://github.com/jrnold/lstbayes){target="_blank"}</td>
-  </tr>
-</table>
+
+*Access Stan's computational backend for advanced applications and development.*
+
+| **Language**           | **Tool** | **Description** |
+|------------------------|----------|-----------------|
+| **C++**                | [**Stan Math Library**](https://mc-stan.org/math/) | Automatic differentiation and mathematical functions used by Stan. |
+| **Multiple Languages** | [**BridgeStan**](https://roualdes.github.io/bridgestan/latest/) | Library providing bindings to a model's log densities, gradients, and more for C++, Python, Julia, R, and Rust. |
+| **R**                  | [**rstantools**](https://mc-stan.org/rstantools/) | Tools for developers of R packages interfacing with Stan. |
+
+---
+
+## Editor and IDE Support
+
+*Enhance your coding experience with Stan language support in your favorite editor.*
+
+| **Editor/IDE**         | **Tool** | **Description** |
+|------------------------|----------|-----------------|
+| **RStudio**            | **Built-in Support** | RStudio 1.2+ includes Stan syntax highlighting and code snippets. |
+| **Visual Studio Code** | [**Stan VSCode Extension**](https://marketplace.visualstudio.com/items?itemName=stan-dev.stan-vscode) | Stan language support with syntax highlighting, autocompletion, and snippets. |
+| **Emacs**              | [**stan-mode**](https://github.com/stan-dev/stan-mode) | Major mode for Stan with syntax highlighting and indentation. |
+| **Vim**                | [**stan-vim**](https://github.com/eigenfoo/stan-vim) | Syntax highlighting, indentation, and code folding for Stan in Vim. |
+| **Atom**               | [**language-stan**](https://github.com/stan-dev/atom-language-stan) | Stan language support in Atom editor. |
+| **Sublime Text**       | [**SublimeStan**](https://github.com/djsutherland/sublime-stan) | Syntax highlighting for Stan in Sublime Text. |
+| **JupyterLab**         | [**jupyterlab-stan-highlight**](https://www.npmjs.com/package/jupyterlab-stan-highlight) | Syntax highlighting for Stan code blocks in JupyterLab. |
+| **Markdown Editors**   | **Pandoc**, **Pygments** | Stan syntax highlighting in markdown via [Pandoc](https://pandoc.org/) and [Pygments](https://pygments.org/). |
+| **LaTeX**              | [**lstbayes**](https://github.com/jrnold/lstbayes) | LaTeX listings for Stan syntax highlighting. |
+
```

Comment:
This table dropped the mentions of Highlight.js and Prism from the current site -- intentional?